app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -110,6 +123,52 @@
 		</div>
 		<div id="map">
 		</div>
+		<div class="modal fade" id="suggestionModal" tabindex="-1" role="dialog" 
+			aria-labelledby="myModalLabel" aria-hidden="true">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal" 
+							aria-hidden="true">x
+						</button>
+						<h4 class="modal-title" id="myModalLabel">
+							Update Information
+					</h4>
+					</div>
+					<div class="modal-body">
+						<form class="form" id="suggestionForm" method="POST" action="/add-suggestion">
+							{!! csrf_field() !!}
+							@include('pages.validation_messages', array('errors'=>$errors))	
```

Comment:
When you use AJAX to send this form data to the server, replace 

```@include('pages.validation_messages', array('errors'=>$errors))``` 

with something like 

```<div class="validation-or-success-messages alert alert-danger"></div>```.  

That element would become where you show your validation messages or success messages.